services/containers/local.go

```diff
@@ -109,8 +111,14 @@ func (l *local) ListStream(ctx context.Context, req *api.ListContainersRequest,
 }
 
 func (l *local) Create(ctx context.Context, req *api.CreateContainerRequest, _ ...grpc.CallOption) (*api.CreateContainerResponse, error) {
+	span := trace.SpanFromContext(ctx)
+	_, ctrCreateSpan := (span.Tracer().Start(ctx, "ctrCreateSpan"))
```

Comment:
The `Context` returned here should be retained and used henceforth in this method so that appropriate parent/child relationships can be established and you don't end up with everything either siblings of a root span or all separate traces.